toolbox/fdc3-workbench/vite.config.ts

```diff
@@ -14,6 +14,7 @@ export default defineConfig({
     outDir: 'build',
   },
   define: {
+    'process.env.NODE_DEBUG': 'false',
```

Comment:
This might be interpreted as `true`, examples I saw had boolean value `false`:
```suggestion
    'process.env.NODE_DEBUG': false,
```

However, we might be fine with setting `true`, its a development tool and its probably just enabling debug output.

Reply:
Actually, example I saw used this syntax, but its not something I've tested in a couple of years:

```
define: {
  'process': {
    env: {
      NODE_DEBUG: false
    }
  }
}
```

Reply:
I tested this and it works

Reply:
you can try loading it on the preview site

Reply:
No I can't unfortunately.

It works because the value is defined, I just suspect its being interpreted as `true`